### ISSUE TITLE
Screen Reader: Settings-Profile: Buttons are announced with redundant 'Floating Action' text

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76947]
## Action Performed:
Prerequisite:
- the user is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/profile
2. Press Tab key to focus the buttons 'Scan receipt' and 'Start chat' at the bottom left corner.
3. Note how the buttons are announced.

## Expected Result:
When users navigate through the settings page, all interactive elements should be announced with a clear and descriptive name, e.g.:
- 'Scan receipt'
- 'Start chat'

## Actual Result:
The buttons are announced with redundant text 'floating action':
- 'Scan receipt (Floating action)'
- 'Start chat (Floating action)'

HTML:
<button aria-label="Scan receipt (Floating action)" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-6koalj r-1777fci r-n7gxbd r-9ffhgg r-lrvibr" data-testid="floating-receipt-button" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-1awozwy r-18og6j9 r-1777fci r-eu3ka r-1aockid" data-testid="floating-receipt-button-container" style="border-radius: 999px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="16" height="16" hovered="false" pressed="false" fill="#FFFFFF"><path d="M3 1.4v17.3c0 .4.5.5.7.3l.7-1q.3-.3.6 0l1.1 1.3q.3.3.6 0L7.9 18q.3-.3.6 0l1.1 1.3q.3.3.6 0l1.1-1.3q.3-.3.6 0l.3.3c-.1-.4-.2-.8-.2-1.3v-.5h-.5c-1.7 0-3-1.3-3-3s1.3-3 3-3h.5V10c0-1.7 1.3-3 3-3 .8 0 1.5.3 2 .8V1.4c0-.4-.5-.5-.7-.3l-.7.9q-.3.3-.6 0L13.9.7q-.3-.3-.6 0L12.1 2q-.3.3-.6 0L10.4.7q-.3-.3-.6 0L8.6 2c-.2.1-.5.1-.7 0L6.8.6q-.3-.3-.6 0L5.1 2c-.2.1-.5.1-.7 0l-.7-.9C3.5.8 3 1 3 1.4"></path><path d="M15 9c.6 0 1 .4 1 1v2.5h2.5c.6 0 1 .4 1 1s-.4 1-1 1H16V17c0 .6-.4 1-1 1s-1-.4-1-1v-2.5h-2.5c-.6 0-1-.4-1-1s.4-1 1-1H14V10c0-.6.4-1 1-1"></path></svg></div></div></button>

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Remove the redundant '(Floating Action)' text from the aria-label.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/a979c941-17b0-4aac-a671-2ab7081d2eeb

<img width="1916" height="998" alt="Image" src="https://github.com/user-attachments/assets/35d70cba-94ad-44c0-9a1f-e54b94818f74" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994988
**Issue reported by:** NA
**Slack conversation:** NA
